lib/api/src/grpc/proto/collections.proto

```diff
@@ -181,6 +183,11 @@ message CollectionParams {
   uint32 shard_number = 3; // Number of shards in collection
   bool on_disk_payload = 4; // If true - point's payload will not be stored in memory
   optional VectorsConfig vectors_config = 5; // Configuration for vectors
+  uint32 replication_factor = 6; // Number of replicas of each shard that network tries to maintain
```

Comment:
why is this field not `optional`?

Reply:
It's only used in the collection info request, that's why shard_number and replication_factor are not options here